monocle/static/js/main.js

```diff
@@ -78,24 +95,47 @@ function getPopupContent (item) {
     } else {
         content += '<br>Disappears in: ' + expires_at + '<br>';
     }
+    content += '<a href="#" data-pokeid="'+item.pokemon_id+'" data-newlayer="Hidden" class="popup_filter_link">Hide</a>';
```

Comment:
Needs a </br> in line with` content += 'Move 2: ' + item.move2 + ' ( ' + item.damage2 + ' dps )'; `otherwise it looks like this after adding iv moveset. Otherwise loving the changes.

![image](https://cloud.githubusercontent.com/assets/9085788/23095423/ac8c1fae-f601-11e6-80c1-7f18d2c55d6c.png)